packages/components/src/Label/_docs/Label.stickersheet.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Label } from "../index"
+
+export default {
+  title: "KAIO-staging/Label",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Header headings={["Default", "Disabled", "Prominent"]} />
+      <StickerSheet.Body>
+        <StickerSheet.Row rowTitle="Base">
```

Comment:
![image](https://github.com/cultureamp/kaizen-design-system/assets/25891850/f8e5e889-58ec-4407-91b4-1d0ae19ff5f8)

Either this or get the headings to shift across one
```suggestion
        <StickerSheet.Row>
```